.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
```

Comment:
Thanks for sharing the project-specific launch.json configuration, it's great to have that set up! However, I noticed it's currently configured for http://localhost:8080, but our app typically runs on port 3000. So This might cause confusion for others. If your local setup **requires** the app to run on port 8080, you can keep it locally and it’s better not to upload it to the shared repository.💙